examples/operator-rbac/2-client.ipynb

```diff
@@ -0,0 +1,918 @@
+{
+ "cells": [
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "## Feast Client with RBAC\n",
+    "### RBAC Kubernetes Authentication\n",
+    "This Feast **Role-Based Access Control (RBAC)** in Kubernetes support authentication  **inside a Kubernetes pod** and **outside a pod** when running a local script.\n",
+    "### Inside a Kubernetes Pod\n",
+    "Feast automatically retrieves the Kubernetes ServiceAccount token from:\n",
+    "```\n",
+    "/var/run/secrets/kubernetes.io/serviceaccount/token\n",
+    "```\n",
+    "This means:\n",
+    "- No manual configuration is needed inside a pod.\n",
+    "- The token is mounted automatically and used for authentication.\n",
+    "- Developer just need create the binding with role and service account accordingly.\n",
+    "- Code Reference:  \n",
+    "[Feast Kubernetes Auth Client Manager (Pod Token Usage)](https://github.com/feast-dev/feast/blob/master/sdk/python/feast/permissions/client/kubernetes_auth_client_manager.py#L15) \n",
+    "- See the example will use service account from pod [Example](https://github.com/feast-dev/feast/blob/master/examples/rbac-remote/client/k8s/)\n",
```

Comment:
This sentence seems confusing, any typo?

Reply:
updated to make it more clear